### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by Steven-Yulong-Yan**
### Motivation

Update position tab fit designs

### Changes

- Add position tab overlay when no position exists
- Add bottom borders
- Flow 2 design

### CODE REVIEW

File: components/Trade/Advanced/index.tsx [hunk 1]
```diff
@@ -10,16 +10,18 @@ const TradingPanel = styled.div`
     width: 25%;
     display: flex;
     flex-direction: column;
-    height: 92vh;
```
Comment (dospore) [inline components/Trade/Advanced/index.tsx:13]: does changing the height of this cause everything to squash up??

Comment (Steven-Yulong-Yan) [reply components/Trade/Advanced/index.tsx:13]: the idea is to reduce the height so it doesn't take up the entire height as in the designs

Comment (dospore) [reply components/Trade/Advanced/index.tsx:13]: ahhhh yeah, just seems to be lots of wasted space. There should be a way to make it to be the full height to allow the children items to fill the space if they need to

**Review by dospore — COMMENTED** [review]
Its looking better, the close position button could do with some padding from the rest of the stuff

File: components/Trade/Advanced/index.tsx (hunk 1 above)
Comment (Steven-Yulong-Yan) [reply components/Trade/Advanced/index.tsx:13]: the footer*

File: components/Buttons/OrderButton/index.tsx [hunk 2]
```diff
@@ -106,9 +106,9 @@ export const PlaceOrderButton: React.FC<POBProps> = ({ className, children }: PO
 };
 
 const CloseOrder = styled(Button)`
-    height: var(--height-small-button);
+    height: var(--height-extra-small-button);
     padding: 0;
-    margin-top: 0.5rem;
+    width: 130px;
```
Comment (dospore) [inline components/Buttons/OrderButton/index.tsx:111]:
```suggestion
    width: 130px;
    margin-left: 0.5rem;
    margin-top: 0.5rem;
```

**Review by dospore — APPROVED** [review]

**Review by dospore — APPROVED** [review]

**Pull request merged**